CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 0.3.5.2 - 2017-06-06
+- Removed log query and add API to check address type [#22](https://github.com/treasure-data/embulk-output-mailchimp/pull/22)
+
```

Comment:
I think it's better to just add tag when you want to release minor version.
Then, include following changes when you release public release version.
* Update CHANGELOG
* Update build.gradle